lib/lookup.json

```diff
@@ -24,6 +24,22 @@
     "prefix": "v",
     "flaky": true
   },
+  "@yarnpkg/cli": {
+    "install": ["install", "--mode", "skip-build"],
+    "maintainers": ["arcanis", "merceyz", "paul-soporan", "larixer"],
+    "prefix": "@yarnpkg/cli@",
+    "repo": "https://github.com/yarnpkg/berry",
+    "scripts": [
+      "build:cli",
+      "typecheck:all",
+      "test:lint",
+      "test:unit",
+      "test:integration"
+    ],
+    "flaky": true,
```

Comment:
I'm not sure this will help, as errors from flaky modules are ignored. We will never detect regressions if `flaky` is true.

Reply:
Timeouts happen every once in a while, but very rarely on Linux.

If increasing the timeout doesn't work, we can add something in our [integration test wrapper](https://github.com/yarnpkg/berry/blob/master/packages/acceptance-tests/pkg-tests-core/sources/utils/tests.ts#L722) to ignore timeouts if a special environment variable if set (by essentially using `Promise.race` to auto-accept a test if it doesn't resolve after some time).

Reply:
The ubuntu error is expected, as Node.js 18 doesn't support Ubuntu 18.04. I opened https://github.com/nodejs/build/issues/2931 for it.

Reply:
> If increasing the timeout doesn't work, we can add something in our [integration test wrapper](https://github.com/yarnpkg/berry/blob/master/packages/acceptance-tests/pkg-tests-core/sources/utils/tests.ts#L722) to ignore timeouts if a special environment variable if set (by essentially using `Promise.race` to auto-accept a test if it doesn't resolve after some time).

Look at the latest CI jobs' logs, I guess we have to try this way then. So what's the name of that env variable, `YARN_TEST_IGNORE_TIMEOUT_FAILURES`? @arcanis

Reply:
Best avoid prefixing it with `YARN_` as those values are reserved for Yarn's settings, but something like `TEST_IGNORE_TIMEOUT_FAILURES` would work 👍